.palantir/revapi.yml

```diff
@@ -743,6 +743,11 @@ acceptedBreaks:
       new: "method java.util.List<org.apache.iceberg.UpdateRequirement> org.apache.iceberg.rest.requests.UpdateTableRequest::requirements()"
       justification: "Signature changed to an interface, but this is safe because\
         \ of type erasure and the original type is always returned"
+  "1.3.0":
+    org.apache.iceberg:iceberg-api:
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.ManageSnapshots org.apache.iceberg.ManageSnapshots::createBranch(java.lang.String)"
```

Comment:
I don't think we want to break the API just by adding a new method. Rather than breaking the API, we typically add a default implementation that throws an UOE, similar to how it's done in `Snapshot#removedDeleteFiles`